packages/react-component-library/src/components/NumberInputE/Button.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import capitalize from 'lodash/capitalize'
+import { IconAdd, IconRemove } from '@defencedigital/icon-library'
+
+import { NUMBER_INPUT_BUTTON_TYPE } from './constants'
+import { StyledButton } from './partials/StyledButton'
+
+type ButtonType =
+  | typeof NUMBER_INPUT_BUTTON_TYPE.DECREASE
+  | typeof NUMBER_INPUT_BUTTON_TYPE.INCREASE
+
+interface ButtonProps {
+  isCondensed: boolean
+  isDisabled: boolean
+  onClick: (event: React.MouseEvent<HTMLButtonElement>) => void
+  type: ButtonType
+}
+
+const svgLookup = {
```

Comment:
`svg` seems like a not too relevant implementation detail of the icons here. How about `iconLookup`?

```suggestion
const iconLookup = {
```